inc/blocks/ab-test/register.php

```diff
@@ -113,6 +113,34 @@ function render_block( array $attributes, ?string $inner_content = '' ) : string
 		$winner = $results['winner'] ?? false;
 	}
 
+	if ( current_user_can( 'edit_posts' ) ) {
+		if ( is_preview() || isset( $_GET['set_test'] ) ) {
+			wp_enqueue_style(
+				'altis-experiments-features-blocks-abtest',
+				plugins_url( 'inc/blocks/ab-test/edit.css', Analytics\ROOT_FILE ),
+				[],
+				null
+			);
+
+			wp_enqueue_script(
+				'ab-test-preview',
+				Utils\get_asset_url( 'blocks/ab-test-preview.js' ),
+				[
+					'wp-i18n',
+				],
+				null
+			);
+
+			return sprintf(
+				'%s<div class="ab-test-xb-preview %s" data-client-id="%s" data-post-id="%s"><div class="ab-test-xb-preview__tabs"></div><div class="ab-test-xb-preview__content"></div></div>',
+				$inner_content,
+				$class_name,
+				$client_id,
+				$post->ID ?? ''
+			);
+		}
+	};
```

Comment:
We don't normally put semicolons after if statements, it's not necessary here at any rate

Reply:
Nasty and bad habit of mine!